src/tests/multihost/sssd/testlib/common/utils.py

```diff
@@ -433,24 +439,24 @@ def join_ad(self, realm=None, adpassword=None, mem_sw=None):
 
     def disjoin_ad(self, realm_output=None, raiseonerr=True):
         """ Disjoin system from Domain """
-        try:
-            self.realm_leave(self.ad_realm, raiseonerr=raiseonerr)
-        except SSSDException:
-            print("Failed to Disjoin system from Windows AD")
         if realm_output:
             account_name = self.get_computer_account(realm_output)
         else:
             account_name = self.multihost.sys_hostname.split('.')[0]
         if len(account_name) > 15:
             account_name = account_name[:15]
-        account_dn = 'CN={},CN={},{}'.format(account_name,
-                                             'Computers',
-                                             self.ad_basedn)
-
-        cmd = "powershell.exe -inputformat none -noprofile "\
-              "'(Remove-ADComputer -Identity"\
-              " \"%s\" -Confirm:$false)'" % (account_dn)
-        self.adhost.run_command(cmd, raiseonerr=False)
+        account_dn = f'CN={account_name},CN=Computers,{self.ad_basedn}'
+
+        # Try to leave the realm normally
+        result = self.realm_leave(self.ad_realm, raiseonerr=False)
+        # Remove the machine on the AD end
+        self.adhost.run_command(
```

Comment:
can you add the command in variable, and call it in run_command to make the command more readable.